pkg/generate/app/sourcelookup.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 var (
-	argumentGit         = regexp.MustCompile("^(http://|https://|git@|git://).*\\.git(?:#([a-zA-Z0-9]*))?$")
+	argumentGit         = regexp.MustCompile("^(http://|https://|git@|git://).*(?:#([a-zA-Z0-9]*))?$")
```

Comment:
In case we ever want to let someone give us a URL to something that isn't a git repo in new-app.